server/auth/auth.go

```diff
@@ -204,14 +204,6 @@ func (a *Authentication) ResetPasswordUser(username string) (types.User, error)
 		return types.User{}, fmt.Errorf("User %q has to be a local user and not a remote one (like LDAP)", username)
 	}
 
-	// TODO : check if JWT can be invalidated.
```

Comment:
Because the reset feature is a **request** for password reset, not the reset itself.

See the content of email (inspired by mailgen):

```
Hi <Toto>,

You have received this email because a password reset request for Docktor account was received.

Click the button below to reset your password:
```